hathor/builder/builder.py

```diff
@@ -145,8 +151,8 @@ def __init__(self) -> None:
         self._enable_tokens_index: bool = False
         self._enable_utxo_index: bool = False
 
-        self._enable_sync_v1: bool = True
-        self._enable_sync_v2: bool = False
+        self._sync_v1_support: SyncSupportLevel = SyncSupportLevel.DISABLED
+        self._sync_v2_support: SyncSupportLevel = SyncSupportLevel.ENABLED
```

Comment:
Are we purposefully changing the default value for each of them? Just checking if this was a mistake since it was not in the acceptance criteria.

Reply:
I'll add to the acceptance criteria, but this is an internal default. Maybe it should have both protocols as unavailable by default. Both builders have a default mechanism that overrides this, tests also have a mechanism that always sets the sync version through the builder and the cli also has a mechanism that sets a version through the builder, so in the end the value here doesn't have any effect on any code.

Reply:
I set both to `SyncSupportLevel.UNAVAILABLE` which is more consistent with all the other properties having an "empty" value as default.

Reply:
I like it. Thanks, Jan!

Reply:
Thanks!